src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonLinkTest.razor

```diff
@@ -0,0 +1,24 @@
+﻿<CascadingValue Value="Visual">
```

Comment:
This approach to resolving the Test issues is not accepted.
please consider the correct solution (resolving the ambiguity of the "string" parameter using @ character) as we talked about in our meeting.